Dockerfile

```diff
@@ -1,35 +1,23 @@
 FROM node:10-alpine as deps
-
 COPY package.json yarn.lock ./
-
-RUN yarn && \
-  yarn install --modules-folder node_modules_prod
+RUN yarn
+RUN yarn install --modules-folder node_modules_prod
 
 # Test
 FROM node:10-alpine as test
-
 RUN mkdir /reflect
```

Comment:
You can remove the `RUN mkdir...` commands here and below because `WORKSPACE` should automatically create dirs that don't exist

Reply:
and done